webpack-test/statsCases/performance-no-hints/c.js

```diff
@@ -1 +1,3 @@
-require.ensure(["./d", "./e"], function(require) {});
+// require.ensure(["./d", "./e"], function(require) {});
+import("./d")
```

Comment:
Sorry, my mistake. I didn't notice that these cases used `require.ensure`. It is not recommended to modify `webpack-test`, these cases are used to test rspack's compatibility with webpack. So, it is more appropriate to copy these cases which need to be modified to `rspack-test-tools/tests`.